src/components/RainWindInfo/WindPrecipitationInfo.tsx

```diff
@@ -52,9 +48,17 @@ const number2HeadTail = (num: number) => {
   return [head, tail];
 };
 
-export const RainWindInfo = ({ rain, windSpeed }: RainWindInfoProps): JSX.Element => {
-  const [windspeedHead, windspeedTail] = number2HeadTail(windSpeed);
-  const [rainProbabilityHead, rainProbabilityTail] = number2HeadTail(rain);
+export type RainPrecipitationInfoProps = {
+  currentWeather: Pick<CurrentWeatherForecast, "wind_speed" | "rain" | "snow" | "temp">;
+};
+
+export const WindPrecipitationInfo = ({ currentWeather }: RainPrecipitationInfoProps): JSX.Element => {
+  const wind = currentWeather.wind_speed;
+  const rain = currentWeather.rain?.["1h"] || 0;
+  const snow = currentWeather.snow?.["1h"] || 0;
```

Comment:
use prop

Reply:
(if actual implementation doesn't fit this usecase - do a simmilar helper - this looks unclean, there is also 1 more (?) place in code afaik - amend that too please